app/javascript/article-form/articleForm.jsx

```diff
@@ -180,9 +173,10 @@ export class ArticleForm extends Component {
   };
 
   failedPreview = (response) => {
-    // TODO: console.log should not be part of production code. Remove it!
-    // eslint-disable-next-line no-console
-    console.log(response);
+    this.setState({
+      errors: response,
+      submitting: false,
+    });
```

Comment:
I don't think this is still quite right, we need to display the errors 🤔  Any thoughts @nickytonline @fdoxyz?

Reply:
That's true, errors are still coming in the `successCb` (`this.showPreview`) rather than `failureCb` (`this.failedPreview`). I guess the catch on the AJAX request only catches the error when its a connection/unexpected error rather than a response with status code other than 200

If we add something to bubble up the error [here](https://github.com/forem/forem/blob/master/app/javascript/article-form/actions.js#L16-L18) to the failureCb then the errros should render correctly in this solution @nickytonline